README.md

```diff
@@ -129,20 +129,21 @@ nationality = user_profile.get('nationality')
 
 ## Running the Examples
 
-The callback URL for both example projects will be http://localhost:5000/yoti/auth/
+The callback URL for both example projects will be ``http://localhost:5000/yoti/auth/``
```

Comment:
Any particular reason why you would use double ``?

Reply:
I misunderstood your comment 
> Please remove the URL and display it as code using ``

I'll change it to use single ones